.phpcs.xml.dist

```diff
@@ -11,7 +11,8 @@
 	<exclude-pattern>resources/*</exclude-pattern>
 
 	<!-- Exclude Composer vendor folder -->
-	<exclude-pattern>vendor/*</exclude-pattern>
+	<exclude-pattern>amp-toolbox/vendor/</exclude-pattern>
+	<exclude-pattern>amp-toolbox-php/vendor/</exclude-pattern>
```

Comment:
This was probably an unintended change.

Reply:
It was intentional, actual. I found that when I ran phpcs it would math no files when I was working with the package inside of the AMP plugin's vendor directory. The first line prevents this from happening. The second line ensures the expected exclusions work on GHA when the repo is cloned standalone.

Reply:
I already merged without it, and I also never saw this problem before. Can you open an issue for me to look into it then?

Reply:
See #132